src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestSubmittedCallbackHandler.java

```diff
@@ -70,7 +70,7 @@ private SscsCaseData updateCase(Callback<SscsCaseData> callback, SscsCaseData ca
     }
 
     private SscsCaseData updateCase(SscsCaseData caseData, long caseId, String eventType, String summary, String description) {
-        var caseDetails = ccdService.updateCase(caseData, caseId, eventType, summary, description, idamService.getIdamTokens());
+        var caseDetails = ccdService.triggerCaseEventV2(caseId, eventType, summary, description, idamService.getIdamTokens());
```

Comment:
This won't persist any modifications to caseData; the caseData parameter is now unused